Build warnings in exception classes

```
/Users/angli/Developer/mobly-bundled-snippets/src/main/java/com/google/android/mobly/snippet/bundled/AccountSnippet.java:55: warning: [serial] serializable class AccountSnippetException has no definition of serialVersionUID
    private static class AccountSnippetException extends Exception {
                   ^
/Users/angli/Developer/mobly-bundled-snippets/src/main/java/com/google/android/mobly/snippet/bundled/BluetoothAdapterSnippet.java:36: warning: [serial] serializable class BluetoothAdapterSnippetException has no definition of serialVersionUID
    private static class BluetoothAdapterSnippetException extends Exception {
                   ^
/Users/angli/Developer/mobly-bundled-snippets/src/main/java/com/google/android/mobly/snippet/bundled/utils/ApiVersionException.java:4: warning: [serial] serializable class ApiVersionException has no definition of serialVersionUID
public class ApiVersionException extends Exception {
       ^
```

--- a/src/main/java/com/google/android/mobly/snippet/bundled/AccountSnippet.java
+++ b/src/main/java/com/google/android/mobly/snippet/bundled/AccountSnippet.java
@@ -53,6 +53,7 @@ public class AccountSnippet implements Snippet {
     private static final String AUTH_TOKEN_TYPE = "mail";
 
     private static class AccountSnippetException extends Exception {
+        private static final long serialVersionUID = 1;
         public AccountSnippetException(String msg) {
             super(msg);
         }

--- a/src/main/java/com/google/android/mobly/snippet/bundled/BluetoothAdapterSnippet.java
+++ b/src/main/java/com/google/android/mobly/snippet/bundled/BluetoothAdapterSnippet.java
@@ -34,6 +34,7 @@ import org.json.JSONException;
 /** Snippet class exposing Android APIs in BluetoothAdapter. */
 public class BluetoothAdapterSnippet implements Snippet {
     private static class BluetoothAdapterSnippetException extends Exception {
+        private static final long serialVersionUID = 1;
         public BluetoothAdapterSnippetException(String msg) {
             super(msg);
         }

--- a/src/main/java/com/google/android/mobly/snippet/bundled/WifiManagerSnippet.java
+++ b/src/main/java/com/google/android/mobly/snippet/bundled/WifiManagerSnippet.java
@@ -39,6 +39,7 @@ import org.json.JSONObject;
 /** Snippet class exposing Android APIs in WifiManager. */
 public class WifiManagerSnippet implements Snippet {
     private static class WifiManagerSnippetException extends Exception {
+        private static final long serialVersionUID = 1;
         public WifiManagerSnippetException(String msg) {
             super(msg);
         }

